docs/quickstart-exit.md

```diff
@@ -0,0 +1,91 @@
+# Clean up
+
+In this document you'll learn how to:
+
+* Exit the MySQL command client shell and the Docker container
+
+* Remove the Docker container and the Docker image
+
+* Remove the Docker volume.
+
+The steps are as follows:
+{.power-number}
+
+1. To exit the MySQL command client shell we use `exit`. You can also use the `\q` or `quit` commands. The execution of the statement also closes the connection.
```

Comment:
Are we exiting the MySQL container or the PXB container? No idea.

Reply:
Added